develop/matchit/expression.h

```diff
@@ -36,6 +36,25 @@ namespace matchit
                            { return v; });
         }
 
+        template <typename T> 
+        constexpr auto toNullary(Nullary<T> const &v)
```

Comment:
Actually, this one is no longer needed, since it will be covered by the general version.

Reply:
You are right, I removed it.